ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/EditorSteps.java

```diff
@@ -69,6 +71,15 @@ public void addAnotherStep(int stepPos) {
         flowViewComponent.clickAddStepLink(stepPos);
     }
 
+    @Then("^verify delete button on step ([0-9]+) (is|is not) visible$")
+    public void verifyDeleteButtonOnStartFinishStepNotVisible(int position, String visibility) throws Throwable {
+        SelenideElement trashIcon = flowViewComponent.getStepOnPosition(position).$(By.className("fa-trash"));
+        assertEquals("Delete icon should" + visibility.replace("is", "") + " be visible",
```

Comment:
when visibility is `is not` it will print "Delete icon shouldnot be visible" :dancer:

Reply:
`java.lang.AssertionError: Delete icon should not be visible expected:<false> but was:<true>`

The `visibility` argument will contain `"is not"`, the `"is"` is removed

Reply:
:man_facepalming: I tried to pretend I'm clever, it turns out I'm not :-(

Reply:
:D anyway, what do you think about such "clever" solutions, is that ok or would a separate step for each case be more readable?

Reply:
I like that, imo it is much better than having 2 methods that are different by only (`assertTrue` vs `assertFalse`) :grin: